cmds/info.c

```diff
@@ -219,7 +220,7 @@ static int fill_cmdline(void *arg)
 	ret = fread(buf, 1, sizeof(fha->buf), fp);
 	fclose(fp);
 
-	if (ret < 0)
+	if (ret != sizeof(fha->buf))
```

Comment:
`ret` is the string length of `buf`, but `sizeof(fha->buf)` is always `4096` so this condition is always true.